docs/content/temporary-components.md

```diff
@@ -0,0 +1,43 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
```

Comment:
@ElleEss should go through the doc before merging it in.

Reply:
Will add as a reviewer.